spec/integration_spec.rb

```diff
@@ -359,167 +355,6 @@
   end
 end
 
-describe 'Book' do
-  before do
-    Book.clear_index!(true)
-    Book.index(safe_index_uid('BookAuthor')).delete_all_documents
-    Book.index(safe_index_uid('Book')).delete_all_documents
-  end
-
-  it 'returns array of tasks on #ms_index!' do
```

Comment:
Moved to `instance_methods_spec.rb` since it concerns the interface (return type) of one of the instance methods we add to models.